### ISSUE TITLE
Convert function - Export to LOR

### ISSUE THREAD

**Issue by Bizywk** [xLightsSequencer/xLights#1820]
Hi guys.  GREAT WORK!     New to xLights and LOVE IT!   We've already begun to phase over from other software.  

Will there be a .loredit convert option in the future? 

In the mean time, I'm asking if you can assist.   PROBLEM: I'm desperately trying to convert a few purchased xLights sequences from .fseq format to .LORedit format for this season, but my results using the convert function are unsuccessful  -   Perhaps because xLights currently only has an option for the outdated .lms format and I must get it to .loredit format?  Could it be that my inexperience with xLights is the issue and I have a setting that is incorrect?   I can't tell and I'm not sure if my results are unique or similar to others, but here's what happens: 

DETAILS: I've tried 3 different sequences so far that all start off rendering and playing flawlessly within xLights.  I close the sequence and attempt to convert it to LOR's .lms format as in intermediate state. It seems to take quite a while to convert the .fseq file and occasionally freezes requiring me to close the software and restart.  About 75% of the time, the xLights software convert function appears to complete the migration to .lms format, but LOR SE v.5 has only accepted those converted files once out of about 30+ tries. 

As I tinker with different export options and approaches, the range of errors I get goes from xLights freezing up during the initial conversion from .fseq, to mostly just a plain LOR's S5 Sequence Editor rejecting the converted sequence as it fails to open the .fseq to .lms converted file.  Sometimes I get as far as LOR asking for me to choose a preview file, but most of the time I get stopped earlier in the process due to a file size error (Below). 

Since I'm a recent convert to xLights, my xLights sequences are paid packages from PPD. While they don't have a huge or complicated layout, I've even tried trimming them down to a single prop prior to converting with mostly the same end results. Only once did the process work, and it was on a stripped down sequence using only one prop. Thankfully, the successful prop was a 16 strand x 50 pixel tree but I can't point to any reason why that was the only time. 

Any Ideas on how I can fix my export problem?  Will there be a .loredit convert option in the future?  

Background details below.

Thanks
Bizywk

Background:
Old LOR guy and New xLights user
Microsoft Windows 10 Pro 64-bit v.1903
Intel® Core™ i7-4770K CPU @ 3.50GHz
NVIDIA GeForce GTX 1070
OpenGL version: 4.6.0 NVIDIA 388.13
xLights v. 2019.64 64b 
Light-O-Rama S5 Sequencer Version 5.3.10 Pro Edition

Most common error: "Error getting size of file E:\Light O Rama\Sequences\Christmas\S5 Sequences\Christmas Vacation\Original\Pixel Pro Displays\ChristmasVac PPD Edits\Arches Only\Christmas Vacation.lms Unable to load the sequence!"

**Comment by AzGilrock** [comment]
I don't think any of us are interested in trying to support exporting to LOR S5 formats.  What we have in xLights worked great to export to S4.

**Comment by Bizywk** [comment]
Not even for undying gratitude and Hero status?       :)

**Comment by Bizywk** [comment]
I just realized that the problem might not be a format problem but actually the export .lms filesize.    

I fired up an old stand alone laptop to load LOR S4 and see if I could at least isolate the problem to the ".lms to .loredit" format conversion.  The export file wouldn't load on my memory stick for some reason.  Then I noticed that the just the 4 arches only .fseq to lms export file is over 5.2Gb.
